crates/transport_iroh/src/lib.rs

```diff
@@ -398,13 +427,18 @@ impl TxImp for IrohTransport {
             };
 
             // Send actual message.
+            let data_len = data.len() as u64;
             send_frame(
                 &connection_context.connection(),
                 FrameType::Payload,
                 data,
             )
             .await?;
 
+            // Update stats
+            connection_context.increment_send_message_count();
+            connection_context.increment_send_bytes(data_len);
```

Comment:
sent bytes are only bytes of the actual message/payload, excluding the frame data.

Reply:
I think it's fine, we get another measure of the bandwidth in wind tunnel by measuring network activity anyway.